packages/starlight-typedoc/tests/unit/typedoc.test.ts

```diff
@@ -67,6 +72,21 @@ test('should generate the doc in `src/content/docs/api` by default', async () =>
   expect(mkdirSyncSpy.mock.calls[0]?.[0].toString()).toMatch(/src[/\\]content[/\\]docs[/\\]api$/)
 })
 
+test('should generate the doc in `/content/docs/api` of the srcDir via the AstroConfig', async () => {
+  await generateTestTypeDoc(
+    {
+      ...starlightTypeDocOptions,
+      entryPoints: ['../../fixtures/basics/src/functions.ts'],
+    },
+    { srcDir: new URL('docs/src', import.meta.url) } as AstroConfig,
```

Comment:
```suggestion
    { srcDir: new URL('www/src', import.meta.url) },
```

And we should be able to remove this cast.

Let's also use `www/src` for the test as `docs` is already part of the path, maybe it's a bit more obvious to use a totally different directory?

Reply:
I just copied my own setup (which gets confusing), so `www` works for me